extension.html

```diff
@@ -117,40 +117,101 @@ <h2>Extensions to the <code>PointerEvent</code> interface</h2>
 };
           </pre>
           <dl data-dfn-for="PointerEvent" data-link-for="PointerEvent">
+            <p>A <a>PointerEvent</a> has an associated <dfn> coalesced event list</dfn> (a list of
+            zero or more <code>PointerEvents</code>). If this event is a <code>pointermove</code>
+            or <code>pointerrawupdate</code> event, it is a sequence of all <code> PointerEvent
+            </code> that were coasesced into this event; otherwise it is the empty list.</p>
+            <p>The events in the <code >coalesced event list </code> will have increasing
+            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>
+            ([[!WHATWG-DOM]]), so the first event will have the smallest <code>timeStamp</code>.
+            </p>
+
+            <p> When a <var>PointerEvent</var> is created, run the following steps for each
```

Comment:
I wonder if you want to move this right after the definition of predicted event list as well.

Reply:
Sounds right, I merge this with the predicted event list one. They are the same.